File: FileManager.API/Models/FileManagerAdmin.cs
using System;
using System.Collections.Generic;

namespace FileManager.API.Models
{
    public class FileManagerAdmin
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public string SubFolderName { get; set; }
        public string FolderData { get; set; }
        public DateTime DateCreated { get; set; }
        public DateTime DateModified { get; set; }       
        public virtual User User { get; set;}
        public virtual ICollection<File> Files { get; set; }
    }
}

File: FileManager-SPA/src/app/_models/filemanageradmin.ts
import { User } from './user';

export interface FileManagerAdmin {
    id: number;
    userId: number;
    subFolderName: string;
    folderData: string;
    dateCreated: Date;
    dateModified: Date;
    user: User;
    companyName: string;
    userName: string;
    email: string;
}
